Makefile

```diff
@@ -138,3 +138,7 @@ clean-proto-defs: | $(clean)
 $(GOBIN_TOOL):
 	go get github.com/myitcv/gobin
 	go install github.com/myitcv/gobin
+
+vpc/service/db/migrations/bindata.go: vpc/service/db/migrations/generate.go vpc/service/db/migrations/*.sql
```

Comment:
Do you want to make `build` rely on this target?

Reply:
Wasn't super sure. It doesn't change that often, but if it isn't in the build target, then who is going to "remember" to run it? :) 

Make should do its thing. I'll add it.